packages/api/test/auth.test.ts

```diff
@@ -18,13 +17,14 @@ const user = 'username';
 const pass = 'changeme';
 
 describe('#auth()', function () {
-  this.beforeAll(function () {
```

Comment:
I'm not really sure how this was working because Mocha doesn't document `beforeAll()` anywhere.